sql/core/src/test/scala/org/apache/spark/sql/DataFrameFunctionsSuite.scala

```diff
@@ -5918,6 +5918,9 @@ class DataFrameFunctionsSuite extends QueryTest with SharedSparkSession {
 
   test("call_function") {
     checkAnswer(testData2.select(call_function("avg", $"a")), testData2.selectExpr("avg(a)"))
+
+    spark.udf.register("default.test_avg", (i: Int) => { i + 2 })
```

Comment:
can we add a test in `HiveUDFSuite` to make sure we can invoke a persist hive function?

Reply:
OK